src/material-experimental/mdc-slider/slider.ts

```diff
@@ -238,6 +256,24 @@ export class MatSlider implements AfterViewInit, OnDestroy {
     return this._isRange() ? [Thumb.START, Thumb.END] : [Thumb.END];
   }
 
+  _throwInvalidInputConfigurationError(missingSelector: string): void {
```

Comment:
Should this be moved out into a function? That way it can be tree-shaken away if we don't use it in production mode.